### DISCUSSION TITLE
Request for Access to Claude AI System Prompts

### DISCUSSION THREAD

**Discussion by aarrumi3** [community/community#136988] [Programming Help]
### Select Topic Area

Question

### Body

Hello,
I recently read about Anthropic's decision to publish the system prompts used to guide the Claude AI models. I'm interested in reviewing these prompts for Claude 3.5 Sonnet or other models in the Claude 3 family. However, I couldn't find them on the official website or apps. Could you please guide me to where these prompts are available, or let me know if they are accessible through a specific GitHub repository or documentation?

**Comment by K3irill** [comment]
https://docs.anthropic.com/en/prompt-library/library
https://docs.anthropic.com/en/release-notes/system-prompts

**Comment by Nuwan-Srimal** [comment]
Anthropic has recently made the system prompts for its Claude 3 family of models, including Claude 3.5 Sonnet, publicly accessible as part of their commitment to transparency. These prompts are available online through Anthropic's official documentation. The prompts provide detailed guidelines that shape the behavior and responses of the Claude models, such as how they handle controversial topics, their approach to coding tasks, and the specific instructions for dealing with images.

To access these prompts, you can visit Anthropic's documentation site, where they have been published. They are available for review and include information on how the models, like Claude 3.5 Sonnet, are instructed to interact with users. This release is part of Anthropic’s broader effort to set a transparency standard in the AI industry.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @aarrumi3 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
